01_install_requirements.sh

```diff
@@ -69,7 +69,6 @@ sudo yum -y install \
   ansible \
   bind-utils \
   jq \
-  libguestfs-tools \
```

Comment:
Is this from something else you were doing?  Not needed here is it?

Reply:
It's removing the requirement as we're not longer doing qemu-img-convert I think?

Reply:
We used to need it for some of the image customisation we did but is no longer needed. Actually I think we might have removed the requirement when we stopped calling virt-filesystems or something.